fixtures.ttar

```diff
@@ -5254,6 +5721,35 @@ Mode: 644
 Directory: fixtures/sys/devices/system/cpu/cpufreq/policy1
 Mode: 755
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Directory: fixtures/sys/devices/system/node
```

Comment:
Looks like some re-ordering is happening here.

Reply:
I double checked, and yes, this is correct ordering.

I guess we should add a repack fixtures test to the CI pipeline.

Reply:
https://github.com/prometheus/procfs/pull/383 should fix this up.

Reply:
I'm not sure what of if I need to fix something here. I forced-pushed after make update_fixtures